Functions.Templates/Templates/DurableFunctionsEntity-JavaScript/metadata.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "DurableFunctionsEntityJS",
```

Comment:
```suggestion
  "defaultFunctionName": "Counter",
```

Would it make more sense to default to `Counter`?

Reply:
Not sure if this was resolved.